backend/src/helpers/pagination/change-pagination-page/change-pagination-page.helper.ts

```diff
@@ -0,0 +1,5 @@
+const changePaginationPage = (page: number): number => {
```

Comment:
I think current naming does not fully describes the purpose of helper, maybe `convertPageToZeroIndexed` or something like that would be more suitable 🤔